openQA-client-test.spec

```diff
@@ -0,0 +1,25 @@
+%define         short_name openQA-client
+Name:           %{short_name}-test
+Version:        4.6
+Release:        0
+Summary:        Test package for %{short_name}
+License:        GPL-2.0-or-later
+BuildRequires:  %{short_name} == %{version}
+ExcludeArch:    i586
+
+%description
+.
+
+%prep
+# workaround to prevent post/install failing assuming this file for whatever
+# reason
+touch %{_sourcedir}/%{short_name}
+
+%build
+/usr/share/openqa/script/client --help
```

Comment:
can you try to just call `openqa-client --help` from PATH?

Reply:
It doesn't seem like a bad idea to me to double-check that it was actually installed in the right place.

Reply:
ok. I copied this from `openQA-worker-test.spec`
So should this be changed/added in that one too?

Reply:
No, because the worker startup script is *not* in PATH

Reply:
@kalikiana I would prefer to use the client from `$PATH` because this is not tested anywhere else and this is what users should use. Checking that everything is "installed in the right place" might sound nice but then you should need to do the same for all other components as well and not scattered like this.

Reply:
changed